### PR TITLE
Chronus changelogs

### PR DESCRIPTION
**Pull request by timotheeguerin**
Issues with changesets versioning that prevent us from using it:
1. This one is a deal breaker bug that bumps to major versions(when we try to bump todays changelog everything gets bumped to 2.0.0) (This is a bug we found happened sometimes in cadl-ranch but this just happens immediately in our repo due to everything being linked)
2. the verify doesn't check all packages have changelog just that there is at least one
3. the `changesets add` doesnt' compare against remote main but local main so you endup with confusing saying other packages changes if you pulled some change from main in your branch but not on your local main

## Example of auto created comment

When changes are needed
image

When everything is documented 
image

## Error is also more readable

image

## `chronus add` experience
image
image
image

### CODE REVIEW

File: .changeset/hot-peaches-refuse.md
Comment (timotheeguerin) [inline .changeset/hot-peaches-refuse.md:1]: remove before merging, just for testing

File: packages/openapi3/README.md
Comment (timotheeguerin) [inline packages/openapi3/README.md:1]: remove before merging, just for testing

File: packages/openapi/README.md
Comment (timotheeguerin) [inline packages/openapi/README.md:1]: remove before merging, just for testing

File: packages/http/README.md
Comment (timotheeguerin) [inline packages/http/README.md:1]: remove before merging, just for testing

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**